Makefile

```diff
@@ -37,6 +37,7 @@ fig:
 
 .PHONY: lint
 lint:
+	docker pull golangci/golangci-lint:latest
```

Comment:
Is this needed to ensure we're using a newer version of the linter moving forward?